tests/global_config.py

```diff
@@ -373,23 +374,23 @@ def _get_default_storage_class(sc_list):
     {
         "rhel-8": {
             DV_SIZE_STR: Images.Rhel.DEFAULT_DV_SIZE,
-            INSTANCE_TYPE_STR: "u1.medium",
+            INSTANCE_TYPE_STR: U1_MEDIUM_STR,
             PREFERENCE_STR: "rhel.8",
             DATA_SOURCE_NAME: "rhel8",
         }
     },
     {
         "rhel-9": {
             DV_SIZE_STR: Images.Rhel.DEFAULT_DV_SIZE,
```

Comment:
since we are touching this part of code maybe moving out common ones DV_SIZE_STR: Images.Rhel.DEFAULT_DV_SIZE, and INSTANCE_TYPE_STR: U1_MEDIUM_STR